README.md

```diff
@@ -35,45 +35,13 @@ See [here](https://tum-dev.github.io/eat-api/enums/labels.json).
 
 ### API
 
-The actual API is provided by static JSON files, which can be found in the gh-pages branch of this repository. These files are created through automatic travis builds. You need to structure a link as follows in order to access the API:
+The actual API is provided by static JSON files, which can be found in the gh-pages branch of this repository. These files are created through automatic travis builds. 
+The documentation can be found at: https://tum-dev.github.io/eat-api/swagger-ui
```

Comment:
```suggestion
[![Documentation](https://img.shields.io/badge/docs-online-informational?style=for-the-badge&link=https://tum-dev.github.io/eat-api/swagger-ui)](https://tum-dev.github.io/eat-api/swagger-ui)
The documentation can be found at: https://tum-dev.github.io/eat-api/swagger-ui
```

Has to be updated to `https://tum-dev.github.io/eat-api/docs` in case we decide to go with this one instead.

Reply:
I agree to that!